main.cpp

```diff
@@ -126,6 +127,7 @@ int main(){
             Sleep(0, 500000);
         }
         
+        //why are we setting these?
```

Comment:
I would need to look over the entirety of the code if I wanted to know what I was thinking but its the kinda thing you are probably correct about me being incorrect.